src/server/models/room/ParticipantRoom.ts

```
@@ -112,12 +112,16 @@ export class ParticipantRoom implements Room {
             throw new Error("Runtime error: Override issue - not receiving events in a HubRoom");
         }
 
+        // Check the event is valid.
+        // @ts-ignore
```

hmm, actually - what is this `tsignore` doing?

---

Oh, because it requires a PDU and will break with an LPDU, but I think this should always be a PDU at this point?

---

we should be casting rather than ignoring the error :p

---

Will update it. I want to double check the code paths of how we get here.

---

The callers always pass in a `PDU` as far as I can tell.